include/mlm_proto.h

```diff
@@ -159,17 +163,18 @@ indicates that the message could not be delivered.
 #define MLM_PROTO_CONNECTION_CLOSE          4
 #define MLM_PROTO_STREAM_WRITE              5
 #define MLM_PROTO_STREAM_READ               6
-#define MLM_PROTO_STREAM_SEND               7
-#define MLM_PROTO_STREAM_DELIVER            8
-#define MLM_PROTO_MAILBOX_SEND              9
-#define MLM_PROTO_MAILBOX_DELIVER           10
-#define MLM_PROTO_SERVICE_SEND              11
-#define MLM_PROTO_SERVICE_OFFER             12
-#define MLM_PROTO_SERVICE_DELIVER           13
-#define MLM_PROTO_OK                        14
-#define MLM_PROTO_ERROR                     15
-#define MLM_PROTO_CREDIT                    16
-#define MLM_PROTO_CONFIRM                   17
+#define MLM_PROTO_STREAM_CANCEL             7
```

Comment:
@wangkf1 any specific reason why this couldn't be number 18? This change causes an ABI breakage

Reply:
Sorry about that, I put it there just so it was next to related messages, and once I fixed it for myself it never occurred to me again.